src/path_tracking_pid_local_planner.cpp

```diff
@@ -215,22 +215,22 @@ bool TrackingPidLocalPlanner::computeVelocityCommands(geometry_msgs::Twist & cmd
   if (dt.isZero()) {
     ROS_ERROR_THROTTLE(
       5, "dt=0 detected, skipping loop(s). Possible overloaded cpu or simulating too fast");
-    cmd_vel = geometry_msgs::Twist();
+    auto cmd_vel = geometry_msgs::Twist();
     cmd_vel.linear.x = pid_controller_.getControllerState().current_x_vel;
     cmd_vel.angular.z = pid_controller_.getControllerState().current_yaw_vel;
-    return true;  // False is no use: https://github.com/magazino/move_base_flex/issues/195
+    return cmd_vel;
```

Comment:
Can you re-add this comment, it provides some useful information.
```
// At the first call of computeVelocityCommands we can't calculate a cmd_vel. We could elaborate on this:
// We can't return false because of https://github.com/magazino/move_base_flex/issues/195 so the current velocity is send instead.
```